docs/howto/wallet.md

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 3
+title: Vega Wallet
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import GitPod from './_gitpod.mdx'
+
+# How to use Vega Wallet APIs
+
+## Introduction
+
+To submit orders and other commands on Vega, users need to sign transactions with a private and public key pair. Users on Vega are pseudonymous and are referenced by their public keys. To assist with API integration there is a **REST API service** served from the Vega Wallet software providing signing and key management. 
+
+The Vega Wallet software can store many wallets, and each wallet can store many key pairs. 
+
+Common actions for a wallet include; **logging in**, **retrieving a key pair** and **signing transactions** (such as an order submission). There are several useful how-to guides below that will help with management of keys and signing.
+
+
+## Logging in to an existing wallet
+
+Connect to a Vega Wallet server and log in. This can be either a hosted wallet, one you run yourself locally or stored in the cloud. The resulting `token` will be used to authenticate with the wallet for other actions (such as signing):
+
+<GitPod />
+
+<Tabs groupId="codesamples1">
+<TabItem value="shell-rest" label="Shell (REST)">
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/wallet/wallet.sh#L46-L50
+```
+
+</TabItem>
+<TabItem value="python-rest" label="Python (REST)">
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/wallet/wallet.py#L61-L65
+```
+
+</TabItem>
+</Tabs>
+
+If successful, the response will include:
+
+| Field          |  Description  |
+| :----------------- | :------------- |
+| `token` | A JSON Web Token (JWT) used to provide authentication with methods on the wallet service (pass in headers as Bearer token). |
+
+<details><summary>Example response</summary>
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/wallet/response-examples.txt#L10-L13
+```
+
+</details>
+
+:::info
+For full example code, please visit the [repo on GitHub](https://github.com/vegaprotocol/sample-api-scripts/blob/master/wallet/).
+:::
+
+## Creating a new wallet
+
+Connect to a wallet server to create a new wallet. As mentioned earlier, this can be either a hosted wallet, one you run yourself locally or stored in the cloud. Just like logging in, once a new wallet has been created, a `token` will be returned and it will be used to authenticate with the wallet for other actions (such as signing):
```

Comment:
```suggestion
Choose a wallet server to create a new wallet. As mentioned earlier, this can be either a hosted wallet, one you run yourself locally or stored in the cloud. Once a new wallet has been created, a `token` will be returned and it will be used to authenticate with the wallet for other actions (such as signing):
```